CoreScriptsRoot/Modules/VR/Panel3D.lua

```diff
@@ -689,24 +695,27 @@ end
 
 --Child class, Subpanel
 local Subpanel = {}
-local Subpanel_mt = { __index = Subpanel }
+Subpanel.__index = Subpanel
 function Subpanel.new(parentPanel, guiElement)
-	local self = setmetatable({}, Subpanel_mt)
+	local self = {}
 	self.parentPanel = parentPanel
 	self.guiElement = guiElement
 	self.lastParent = guiElement.Parent
-	self.ancestryConn = nil
-	self.changedConn = nil
+	self.ancestryConn = false
```

Comment:
Ah, I had done that because I was going to add a __newindex that threw an error in case someone typo'd a member name while setting it. Setting it to nil removes it from the table, so you'd never be able to update it later. I'll change it back.